jinja2/base.html

```diff
@@ -16,6 +16,11 @@
     <link rel="copyright" href="#copyright">
   {% endif %}
 
+  {% block site_analytics %}
+    {% javascript 'dnt-helper' %}
+    {% include "includes/google_analytics.html" %}
```

Comment:
Is there an enforced order to scripts in the header? If the GA script block executes before the dnt-helper scripts, then GA won't load, no matter what the user's DNT setting is.

Reply:
So, if I understand things correctly, when the browser gets to the `dnt-helper` it will pause rendering, fetch the script, parse and execute. Only then will it proceed to the next piece of the DOM which would be the GA bits. There are of course optimizations that browsers do but, inside the head I feel confident it will follow the above path.

I tested this throttling the connection all the way down the GPRS and it consistently loads GA if DNT is disabled, and does not, if DNT is enabled.

Reply:
That's my understanding as well 👍